libraries/chain/transaction_context.cpp

```diff
@@ -59,6 +59,7 @@ namespace eosio::chain {
       undo();
       *trace = transaction_trace{}; // reset trace
       initialize();
+      transaction_timer.stop();
```

Comment:
This was not needed. The `undo()` call above calls `transaction_timer.stop()`. I can remove it, but having it here makes it more obvious that it is being called.